app/docs/enterprise/0.31-x/kong-architecture-patterns.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Kong Implementation Checklist
+---
+# Kong Architecture Patterns
+
+An important part of setting up a new Kong cluster is choosing the right architectural pattern that fits your needs and integrates with your existing infrastructure. Here are a few that we commonly see and recommend. Kong itself is greatly flexible, and can be setup in many ways. We present these as starting points in hopes that you find that using or extending one of these patterns provides an ideal fit for your unique needs.
+
+## Kong between client and upstream
+
+The classic architectural pattern we see customers setting up is adding Kong in between some, or all, of their API traffic. Kong is installed in your prefered hosting provider, and configured to proxy traffic to the existing APIs. Note that in this pattern the APIs are still accessible directly, that is both through the proxy and also without being routed through the API Gateway. We see this during initial testing, and migration phases, before customers eventually route all traffic through Kong. 
+
+The pros of this approach are that it is simple to setup and configure. All it requires is setup, and coordinating new traffic to use the gateway. Older traffic will still flow directly to the APIs.
```

Comment:
I would break Pros & Cons into `<h3>` tags to help make them stand out.

Reply:
Noted.